app/src/main/java/chat/rocket/android/renderer/MessageRenderer.java

```diff
@@ -76,10 +75,11 @@ public MessageRenderer timestampInto(TextView textView) {
     }
 
     switch (object.getSyncState()) {
```

Comment:
@leonardoaramaki I agree! NOT_SYNCED seems to be the first state. `MessageInteractor::send` exemplifies that. The state then becomes SYNCING and later either SYNCED or FAILED.